### DISCUSSION TITLE
Do all watchers, eventTap, etc. automatically stop on config reload, or do I need to manually stop them every time?

### DISCUSSION THREAD

**Discussion by minusfive** [Hammerspoon/hammerspoon#3680] [Q&A] [answered]
Wondering whether I should always stop an application.watcher, pathwatcher, eventTap, etc., before reloading a config, or whether it'll automatically be stopped / destroyed every time the config is reloaded by default?

**Comment by cmsj** [comment, marked as answer]
Everything you create in Lua should be destroyed for you when the config is reloaded.
